src/compute-pr-actions.ts

```diff
@@ -487,12 +488,12 @@ function createWelcomeComment(info: ExtendedPrInfo, post: (c: Comments.Comment)
     display(``,
             `## Status`,
             ``,
-            ` * ${emoji(!info.hasMergeConflict)} No merge conflicts`);
+            ` * ${emoji.failure(!info.hasMergeConflict)} No merge conflicts`);
```

Comment:
This is kind of confusing to read, would be better if the argument was flipped, so `emoji.failure(x)` means failure if `x` is true.

Reply:
I ended up switching to `.passed` to match up with the other functions. But yes 😄 thanks!